jlm/llvm/opt/alias-analyses/Andersen.hpp

```diff
@@ -25,10 +25,41 @@ class Andersen final : public AliasAnalysis
   class Statistics;
 
 public:
-  Andersen() = default;
+  /**
+   * class for configuring the Andersen pass, such as what solver to use.
+   */
+  class PassConfiguration
```

Comment:
Just a suggestion, but maybe `Configuration` is enough? It would then be `Andersen::Configuration`, which is quite readable.